grafana-plugin/src/components/Modal/Modal.tsx

```diff
@@ -66,7 +66,7 @@ const getStyles = (theme: GrafanaTheme2) => {
       padding: 15px;
       background: ${theme.colors.background.primary};
       border: 1px solid ${theme.colors.border.weak};
-      box-shadow: var(--shadows-z3);
+      box-shadow: ${theme.isLight ? '0 13px 20px 1px rgba(24, 26, 27, 0.18)' : '0 8px 24px rgb(1, 4, 9)'};
```

Comment:
Why do we need this change since it seems `var(--shadows-z3)` holds exactly the same values?

Reply:
`--shadows-z3` doesn't exist

Reply:
Hm but I can see it's there, e.g.
<img width="1365" alt="image" src="https://github.com/user-attachments/assets/bdc761d0-5755-4dc0-8e8a-25c58d2f3e9b">

Reply:
We can probably access it with `${theme.shadows.z3}` as well, could you try?

Reply:
Oh right, there's a `shadows` field in there. good catch, I changed it

Reply:
Though not all box-shadows in oncall match the defined shadows in grafana, e.g. SortableContainer

```jsx
box-shadow: ${theme.isDark ? 'rgb(17 18 23) 0 0 0 2px, rgb(61 113 217) 0 0 0 4px;' : ''};
background: ${theme.isDark ? '#34363d' : ''};
```